packages/@react-aria/select/docs/useSelect.mdx

```diff
@@ -277,17 +322,24 @@ function Option({item, state}) {
     </li>
   );
 }
-
-<Select label="Favorite Color">
-  <Item>Red</Item>
-  <Item>Orange</Item>
-  <Item>Yellow</Item>
-  <Item>Green</Item>
-  <Item>Blue</Item>
-  <Item>Purple</Item>
-</Select>
 ```
 
+</details>
+
+## Styled examples
+
+<ExampleCard
+  url="https://codesandbox.io/s/hardcore-moon-xzc4r?file=/src/ComboBox.tsx"
+  preview={examplePreview}
+  title="Tailwind CSS"
+  description="An example of styling a Select with Tailwind." />
```

Comment:
Will dig later, but just wanted to let you know that the `Select` component in this sandbox doesn't allow you to navigate the menu via the arrow keys. Pressing up/down arrow just opens and closes the menu but the `Select`s in the other sandboxes work just fine

Reply:
Yeah there's a conflict between shouldUseVirtualFocus being set to true to make combobox work but that breaks select. Should be fixed when I upgrade to the nightly that has my api changes in it which make the user not have to pass that in themselves.

Reply:
Gotcha, that makes sense. Overall looks good to me then